src/Panel/View.php

```diff
@@ -275,6 +275,9 @@ public static function globals(): array
 
 		return [
 			'$config' => fn () => [
+				'api'         => [
+					'methodOverwrite' => $kirby->option('api.methodOverwrite', false)
```

Comment:
I remember that quite a lot of shared hosting environments disable patch requests for some reason. I think we should keep them off by default tbh.

Reply:
I really don't mind either or. I changed it because @lukasbestle said the opposite of you in the forum 😄 no idea myself who is right